provisioning/local/aws.tf

```diff
@@ -1,6 +1,6 @@
 provider "aws" {
   region  = local.region
-  profile = var.aws_profile
```

Comment:
Myslel jsem, ze jsme se shodli na nastaveni account_id, ne profilu? :thinking: V principu si pouzij jakykoliv profil chces/mas nakonfigurovanu, dokud ma pristup k patricnymu accountu.
https://github.com/keboola/sandboxes-service/blob/main/provisioning/local/aws.tf#L2

Reply:
No to si nemyslim, minimalne ja jsem celou dobu mluvil o profilech.  A vychazel jsem i z navodu v conflu a predpokladal ze vsichni mame nove pojmenovany ty profily uplne stejne, tak jako jsou v navodech v confluence.

Reply:
Pro klid duse tam muzu pridat klidne `allowed_account_ids` a tam dat ID accountu. Nicmene ten profil bych tam nechal, protoze ten novy naming bysme meli mit stejny.